share/functions/__fish_portage_print_installed_pkgs.fish

```diff
@@ -0,0 +1,3 @@
+function __fish_portage_print_installed_pkgs --description 'Print all installed packages'
+	find /var/db/pkg -mindepth 2 -maxdepth 2 -type d -printf '%P\n' | sed 's/-[0-9][0-9.]*.*$//' | sort -u
```

Comment:
fish will take care of the deduplication for you - no need to use `sort`.

Also we prefer our builtins over external commands like `sed`. Something like

```fish
find ... | string replace -r -- '-[0-9][0-9.]*.*$' ''
```

should work.

Reply:
Fish's deduplication will only be done if the function is used as input for completion, correct?

Reply:
Yes. What else do you want to use this for?

Reply:
Well, there is __fish_print_packages, however I didn't quite understand what the purpose of that function is/was.

Reply:
Completions. It prints packages for a variety of package managers, so you can theoretically have a tool that takes a name of a system package and complete it just by calling that function.

I don't think that exists, and I've wanted to split it up for a while - because right now if you e.g. have `pacman` installed (e.g. to create an archlinux container) and try to complete `yum` it'll suggest pacman's package names, because that's first in `__fish_print_packages`.